stable/datadog/templates/container-system-probe.yaml

```diff
@@ -0,0 +1,27 @@
+{{- define "system-probe-agent" -}}
```

Comment:
For the sake of consistency, could you rename this to `container-system-probe` so it matches the file name?